embedded-graphics/src/primitives/triangle.rs

```diff
@@ -411,130 +384,82 @@ where
 mod tests {
     use super::*;
     use crate::pixelcolor::BinaryColor;
-    use crate::unsignedcoord::UnsignedCoord;
 
     #[test]
     fn dimensions() {
         let tri: Triangle<BinaryColor> =
-            Triangle::new(Coord::new(5, 10), Coord::new(15, 20), Coord::new(5, 20));
-        let moved = tri.translate(Coord::new(-10, -10));
-
-        assert_eq!(tri.p1, Coord::new(5, 10));
-        assert_eq!(tri.p2, Coord::new(15, 20));
-        assert_eq!(tri.p3, Coord::new(5, 20));
-        assert_eq!(tri.size(), UnsignedCoord::new(10, 10));
-
-        assert_eq!(moved.p1, Coord::new(-5, 0));
-        assert_eq!(moved.p2, Coord::new(5, 10));
-        assert_eq!(moved.p3, Coord::new(-5, 10));
-        assert_eq!(moved.size(), UnsignedCoord::new(10, 10));
+            Triangle::new(Point::new(5, 10), Point::new(15, 25), Point::new(5, 25));
```

Comment:
Did these values need to change? It adds noise to the diff.

Reply:
I changed some of the tests because they were resulting in equal x and y coordinates. Having different results for x and y makes it easier to spot if x and y where mixed up somewhere.

Reply:
Ah I see. Good catch, thank you!